pyquil/operator_estimation.py

```diff
@@ -913,56 +927,50 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
                       DeprecationWarning)
         symmetrize_readout = readout_symmetrize
 
+    if symmetrize_readout is None:
+        symmetrize_readout = SymmetrizationLevel.NONE
+        warnings.warn("'symmetrize_readout' should now be an int, 0 instead of none.",
+                      DeprecationWarning)
+    elif symmetrize_readout == 'exhaustive':
+        symmetrize_readout = SymmetrizationLevel.EXHAUSTIVE
+        warnings.warn("'symmetrize_readout' should now be an int, -1 instead of 'exhaustive'.",
+                      DeprecationWarning)
+    elif symmetrize_readout not in {level.value for level in SymmetrizationLevel}:
+        raise ValueError("'symmetrize_readout' must be an int from -1 to 3 inclusive.")
```

Comment:
I really think you should use the enum to list out the possibilities as in my example -- not only does it give you some sense as two what the numbers mean, but also it will continue to work if you change the enum